modules/base/admin/module.py

```diff
@@ -217,7 +217,8 @@ async def repository_install(self, ctx, url: str, branch: Optional[str] = None):
         await ctx.send(
             _(
                 ctx,
-                "Repository has been installed to `{path}`. It includes the following modules: {modules}.",
+                "Repository has been installed to `{path}`. It includes the following" 
+                "modules: {modules}.",
```

Comment:
A space is missing between those two lines. It would be better if it was split by the sentence:
```py
"Repository has been installed to `{path}`. "
"It includes the following modules: {modules}.",
```
(note the space before the first string ends)